Package.swift

```diff
@@ -8,11 +8,13 @@ let package = Package(
         .library(name: "OpenTelemetryXRay", targets: ["OpenTelemetryXRay"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/slashmo/swift-otel.git", .upToNextMinor(from: "0.8.0")),
+        .package(url: "https://github.com/apple/swift-nio.git", from: "2.0.0"),
```

Comment:
Nitpick here: Can we move the NIO dependency below the OTel dependency since OTel is the primary one?